modules/docs/arrow-docs/docs/docs/generic/product/README.md

```diff
@@ -96,12 +96,12 @@ hListOf(1000, 900).toAccount()
 In the examples below we can observe how 2 different `Int` properties are returned inside a type constructor such as `Option`, `Try`, `Deferred` etc... and the automatically mapped to the shape of our `Account` data class removing all boilerplate from extracting the values from their context and returning an `Account` value in the same context.
 
 ```kotlin:ank
-import arrow.instances.*
+import arrow.core.extensions.option.applicative.applicative
 
 val maybeBalance: Option<Int> = Option(1000)
 val maybeAvailable: Option<Int> = Option(900)
 
-ForOption extensions { 
+Option.applicative().run { 
```

Comment:
You should be able to call `Option.applicative().mapToAccount(maybeBalance, maybeAvailable)` directly here.

Reply:
`generic` does not export extensions for all the applicative data types, just polymorphic functions that can be used in the scope of applicative.